### PR TITLE
Properties: Handle edge case

### PR DESCRIPTION
**Pull request by fdiaz**
We were not handling the 

```swift
let red, green, blue: Double
let seconds: Int, hours, years: Double
```

cases correctly.

This PR adds unit tests + handles this use case.

### CODE REVIEW

File: Sources/SwiftInspectorVisitors/PropertySyntaxVisitor.swift [hunk 1]
```diff
@@ -102,6 +92,19 @@ public final class PropertySyntaxVisitor: SyntaxVisitor {
 
     return modifier
   }
+
+  private func findTypeName(from node: TypeAnnotationSyntax?) -> String? {
+    guard
```
Comment (fdiaz) [inline Sources/SwiftInspectorVisitors/PropertySyntaxVisitor.swift:97]: This is basically a copy/paste from https://github.com/fdiaz/SwiftInspector/pull/99/files#diff-8622b0ddc6698d01f425675d628fce88a76b661f30c567a9756e1282c9981e87L38

File: Sources/SwiftInspectorVisitors/PropertySyntaxVisitor.swift [hunk 2]
```diff
@@ -33,30 +33,20 @@ public final class PropertySyntaxVisitor: SyntaxVisitor {
   public override func visit(_ node: VariableDeclSyntax) -> SyntaxVisitorContinueKind {
     let modifier = findModifiers(from: node)
 
-    node.bindings.forEach { binding in
+    var lastFoundType: String?
+    node.bindings.reversed().forEach { binding in
```
Comment (fdiaz) [inline Sources/SwiftInspectorVisitors/PropertySyntaxVisitor.swift:37]: We're starting from the bottom ~now we're here~ to find the last type found.

File: Sources/SwiftInspectorVisitors/PropertySyntaxVisitor.swift [hunk 3]
```diff
@@ -33,30 +33,20 @@ public final class PropertySyntaxVisitor: SyntaxVisitor {
...
-        // TODO: Include logic to get types of any variable declaration
+        let typeName = findTypeName(from: binding.typeAnnotation)
+        // If we find a type name on the n element, but n-1 doesn't have an explicit type name, n-1 has the same type as n
+        // e.g. let red: Int, green, blue: Double
+        // where both green and blue are of type Double
+        if let typeName = typeName { lastFoundType = typeName }
         propertiesInfo.insert(.init(
                                 name: identifier.identifier.text,
-                                typeAnnotation: nil,
+                                typeAnnotation: typeName ?? lastFoundType,
```
Comment (dfed) [inline Sources/SwiftInspectorVisitors/PropertySyntaxVisitor.swift:46]: since we do `if let typeName = typeName { lastFoundType = typeName }` on the line above, can't we just do:
```suggestion
                                typeAnnotation: lastFoundType,
```

**Review by dfed — APPROVED** [review]
